### ISSUE TITLE
Replace bioversions dependency

### ISSUE THREAD

**Issue by jsstevenson** [cancervariants/disease-normalization#141]
I was trying to be slick, but I think this is a heftier dependency than what we need for something this simple -- we should just swap it out for basic API calls

**Comment by korikuzma** [comment]
I think we should bump the priority on this: 

disease-normalizer==0.4.0.dev0
bioversions==0.5.269

```
$ python3 -m disease.cli --sources="oncotree"
.venv/lib/python3.11/site-packages/pydantic/_internal/_config.py:269: UserWarning: Valid config keys have changed in V2:
* 'underscore_attrs_are_private' has been removed
  warnings.warn(message, UserWarning)
* Owlready2 * Warning: optimized Cython parser module 'owlready2_optimized' is not available, defaulting to slower Python implementation
***Using Disease Database Endpoint: http://localhost:8000***

Deleting OncoTree...
Deleted OncoTree in 0.02974 seconds.

Loading OncoTree...
Traceback (most recent call last):
  File ".venv/lib/python3.11/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File ".venv/lib/python3.11/site-packages/disease/cli.py", line 295, in <module>
    update_db()
  File ".venv/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/disease/cli.py", line 291, in update_db
    _update_sources(sources_to_update, db, update_merged, from_local)
  File ".venv/lib/python3.11/site-packages/disease/cli.py", line 132, in _update_sources
    _load_source(n, db, delete_time, processed_ids, from_local)
  File ".venv/lib/python3.11/site-packages/disease/cli.py", line 182, in _load_source
    processed_ids += source.perform_etl(use_existing=from_local)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/disease/etl/base.py", line 45, in perform_etl
    self._extract_data(use_existing)
  File ".venv/lib/python3.11/site-packages/disease/etl/base.py", line 185, in _extract_data
    self._data_file = self._get_latest_data_file()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/disease/etl/base.py", line 154, in _get_latest_data_file
    self._version = self.get_latest_version()
                    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/disease/etl/base.py", line 59, in get_latest_version
    return bioversions.get_version(self._src_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/sources/__init__.py", line 170, in get_version
    return resolve(name).version
           ^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/sources/__init__.py", line 152, in resolve
    return _resolve_helper_cached(name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/cachier/core.py", line 312, in func_wrapper
    return _calc_entry(core, key, func, args, kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/cachier/core.py", line 75, in _calc_entry
    func_res = func(*args, **kwds)
               ^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/sources/__init__.py", line 159, in _resolve_helper_cached
    return _resolve_helper(name)
           ^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/sources/__init__.py", line 165, in _resolve_helper
    return getter.resolve()
           ^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/utils.py", line 200, in resolve
    version=cls.version,
            ^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/utils.py", line 84, in version
    if isinstance(cls._cache_prop, str):
                  ^^^^^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/utils.py", line 78, in _cache_prop
    cls._cache = cls().get()
                 ^^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/bioversions/sources/oncotree.py", line 26, in get
    res_json = res.json()
               ^^^^^^^^^^
  File ".venv/lib/python3.11/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

https://oncotree.mskcc.org/api/version gets 404

**Comment by jsstevenson** [comment]
OPE closed in #163
